### ISSUE TITLE
update-ref-docs-dry-run_istio doesn't test anything until its already broken

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#23475]
https://github.com/istio/istio/pull/23437 broke the reference docs, but the test passes. I think its because the test pulls istio/istio from master

**Comment by clarketm** [comment]
I am waiting for confirmation from @ericvn in https://github.com/istio/test-infra/pull/2607, but I think the plan is to remove this test from `istio/istio`.

**Comment by ericvn** [comment]
I responded that I personally would like to make the check required in istio/istio to prevent the failing PRs from ever getting in that caused this.

**Comment by ericvn** [comment]
Of course making it required right now would halt the pipeline. I do have the PR in istio/istio.i, https://github.com/istio/istio.io/pull/7212 to fix the problem, but waiting on a docs to approve.

**Comment by ericvn** [comment]
I think this issue is now ready to be closed since the test is now removed.

**Comment by clarketm** [comment]
> I think this issue is now ready to be closed since the test is now removed.

I suggest we close this and if we want to create an issue for _changing the test to support running against the current commit_ then we can create a new issue and assign it to me. (https://github.com/istio/test-infra/pull/2607#issuecomment-623475690)
